tracer/src/Datadog.Tracer.Native/debugger_probes_instrumentation_requester.cpp

```diff
@@ -352,13 +365,25 @@ void DebuggerProbesInstrumentationRequester::AddLineProbes(
         {
             const DebuggerLineProbeDefinition& current = lineProbes[i];
 
+            if (IsProbeIdExist(current.probeId))
+            {
+                Logger::Info("[AddLineProbes] Method Probe Id: ", current.probeId, " is already processed.");
```

Comment:
Should this be `DEBUG`?

Reply:
Done in f2e39f19694d2e9a44c777ae15e0fa38a3d73526.